Test/Unit/Block/JsTest.php

```diff
@@ -249,7 +256,8 @@ public function __construct_always_setsInternalProperties()
             $this->checkoutSessionMock,
             $this->cartHelperMock,
             $this->bugsnagHelperMock,
-            $this->deciderMock
+            $this->deciderMock,
+            $this->eventsForThirdPartyModules
         );
         static::assertAttributeEquals($this->configHelper, 'configHelper', $instance);
```

Comment:
**We should test the $eventsForThirdPartyModules argument. Something like this:**
 
static::assertAttributeEquals($this-> eventsForThirdPartyModules, 'eventsForThirdPartyModules', $instance);

Reply:
added